servers/auth-server/src/main/webapp/WEB-INF/ups-realm.json

```diff
@@ -61,9 +61,9 @@
             "name": "unified-push-server-js",
             "enabled": true,
             "publicClient": true,
-            "baseUrl": "/ag-push",
+            "baseUrl": "http://localhost:8082/ag-push",

```

Comment:
I think this change is not needed. If auth-server is used it will be always "embedded" with UPS. @matzew @abstractj  wdyt ?